apps/financial-aid/backend/migrations/20211109165922-staff-alter-roles-enum.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      queryInterface
+        .removeColumn('staff', 'roles', { transaction: t })
+        .then(() =>
+          queryInterface.sequelize.query(
+            `DROP TYPE enum_staff_role;
+            CREATE TYPE enum_staff_roles AS ENUM ('Employee','Admin','SuperAdmin');
+            ALTER TABLE staff ADD COLUMN roles enum_staff_roles[] NOT NULL DEFAULT '{Employee}';`,
```

Comment:
This enum was wrongly named, hence the migration, will update the staff table on dev once this is merged